packages/template-cli/README.md

```diff
@@ -26,67 +26,111 @@ Creates an amidostacks template in your local directory much like express-genera
 
 1. Install dependencies required to generate the templated app:
 ```bash
-cd ${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/packages/template-cli
-npm install
-npm run build
+$ cd ${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/packages/template-cli
+$ npm install
+$ npm run build
 ```
 
-2. Change to a directory for testing - e.g. $HOME and run
+2. Watch your build
 ```bash
-cd ${SELECTED_DIR}
-npx ${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/packages/template-cli
+$ cd ${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/packages/template-cli
+$ npm run build:watch
 ```
 
-3. Follow the CLI prompts to create a templated app with user defined configuration:
-- Project Name: _default_: current directory (will create a directory with this name)
-- Project Type: _default_: SSR (server side rendering)
-- Platform: _default_: AKS (Azure Kubernetes Service)
-- CI/CD Tooling: _default_: AzureDevOps
+3. Change to a directory for testing - e.g. $HOME and run
+```bash
+$ cd ${SELECTED_DIR}
+$ npx ${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/packages/template-cli/dist/index.js run -i
+```
 
-4. Navigate to the created project directory to install template dependencies:
+4. Follow the CLI prompts to create a templated app with user defined configuration:
+   - Project Name: _default_: current directory (will create a directory with this name)
+   - Project Type: _default_: SSR (server side rendering)
+   - Platform: _default_: AKS (Azure Kubernetes Service)
+   - CI/CD Tooling: _default_: AzureDevOps
+
+5. Navigate to the created project directory to install template dependencies:
 e.g. ${SELECTED_DIR}/$PROJECT_NAME
 ```bash
-cd ${SELECTED_DIR}/$PROJECT_NAME/src/ssr
-npm install
+$ cd ${SELECTED_DIR}/$PROJECT_NAME/src/
+$ npm install
 ```
 
-5. Build and run the app:
+6. Build and run the app:
 ```bash
-npm run build
-npm run start
+$ npm run build
+$ npm run start
 ```
 Open Browser and hit [http://localhost:3000](http://localhost:3000)
 
-6. To test the deploy folder has been correctly provisioned prior to checking you need to at this point in time copy over a sample backend-config and terraform vars. Currently vars.tf and provider configuration is not automatically updated. 
+7. To get a local deployment open the following files and uncomment these lines
+
+`${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/deploy/terraform/azure/provider.tf`
+```terraform
+   # resource_group_name  = "amido-stacks-rg-uks"
+   # storage_account_name = "amidostackstfstategbl"
+   # container_name       = "tfstate"
+   # # key                  = "spa-cycle-2"
```

Comment:
we should clean this up a bit more - correct instructions can be found in the `libs/orchestration/terraform-azurerm-amido-aks`

as they are all currently being updated it might be worth revisiting this a bit later

Reply:
removed